### PR TITLE
adding gmail type 100 with delaying of 100ms case

### PR DESCRIPTION
**Pull request by ypwalter**
@ShakoHo @MikeLien @askeing r?

### CODE REVIEW

File: tests/regression/gmail/test_chrome_gmail_ft_type_in_reply_field_100_txt_delay_100ms.sikuli/test_chrome_gmail_ft_type_in_reply_field_100_txt_delay_100ms.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        compare_area = self.tuning_region(reply_btn_region, x_offset=-10, y_offset=30, w_offset=650, h_offset=15)
+        self.set_override_region_settings(customized_region_name_start, compare_area)
+        self.set_override_region_settings(customized_region_name_end, compare_area)
+
+        t1 = time.time()
+        capimg1 = capture(0, 0, capture_width, capture_height)
+
+        char_len = 100
+        sample_str = string.letters + "\n" + string.letters[::-1]
+        sample_str.replace("t", "Q").replace("i", "Q").replace("j", "Q").replace("l", "Q")\
```
Comment (ypwalter) [inline tests/regression/gmail/test_chrome_gmail_ft_type_in_reply_field_100_txt_delay_100ms.sikuli/test_chrome_gmail_ft_type_in_reply_field_100_txt_delay_100ms.py]: Please Refer `https://stackoverflow.com/questions/3411771/multiple-character-replace-with-python` for the reason I chose to replace characters in this way.

**Comment by ypwalter** [comment]
@ShakoHo r?

File: tests/regression/gmail/test_chrome_gmail_ft_type_in_reply_field_100_txt_delay_100ms.sikuli/test_chrome_gmail_ft_type_in_reply_field_100_txt_delay_100ms.py (hunk 1 above)
Comment (ShakoHo) [inline tests/regression/gmail/test_chrome_gmail_ft_type_in_reply_field_100_txt_delay_100ms.sikuli/test_chrome_gmail_ft_type_in_reply_field_100_txt_delay_100ms.py]: Based on the reference, it seems better to use loop list to replace characters?

Comment (ypwalter) [inline tests/regression/gmail/test_chrome_gmail_ft_type_in_reply_field_100_txt_delay_100ms.sikuli/test_chrome_gmail_ft_type_in_reply_field_100_txt_delay_100ms.py]: replace 2 character: method i
replace 17 character: method b

Comment (ShakoHo) [inline tests/regression/gmail/test_chrome_gmail_ft_type_in_reply_field_100_txt_delay_100ms.sikuli/test_chrome_gmail_ft_type_in_reply_field_100_txt_delay_100ms.py]: ok

**Comment by ypwalter** [comment]
thanks

**Pull request merged**